tests/cpptests/micro-benchmarks/benchmark_idlist_iterator.cpp

```diff
@@ -92,15 +90,13 @@ BENCHMARK_DEFINE_F(BM_IdListIterator, Read_Old)(benchmark::State &state) {
 
 BENCHMARK_DEFINE_F(BM_IdListIterator, SkipTo_Old)(benchmark::State &state) {
   IndexIterator *iterator_base = NewIdListIterator(docIds.data(), docIds.size(), 1.0);
-  RSIndexResult *hit;
+  RSIndexResult *hit = iterator_base->current;
```

Comment:
is the current properly set to docID 0? when instantiated? should we add this to the unittest?

Reply:
well not in unit tests, but are we sure?

Reply:
I would do `lastDocID = 0`at the beginning, and then update with `hit`after every call to SkipTo (which I believe that according to interface could be more ensured)?

Reply:
When initializing and rewinding. In the new implementation, I rely on the `lastDocId`, which should be zeroed more strictly.
I can replace the old implementation benchmarks with a more "manual" tracking so we don't have to worry about current implementation behavior